Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageMigrationResponseMapperTest.cs

```diff
@@ -0,0 +1,199 @@
+            var affectedContract = CreateContract();
+            var affectedRelation = CreateSystemRelation();
+            var affectedDpr = CreateDpr();
+
+            var entity = new ItSystemUsageMigration
+            (
+                fromUsage,
+                fromSystem,
+                toSystem,
+                new List<ItContract> {affectedContract},
+                new List<SystemRelation> {affectedRelation},
+                new List<DataProcessingRegistration> {affectedDpr}
+            );
+
+            //Act
+            var result = _sut.MapMigration(entity);
+
+            //Assert
+            AssertIdentityNamePairWithDeactivatedStatus(entity.SystemUsage, result.TargetUsage);
+            AssertIdentityNamePairWithDeactivatedStatus(entity.FromItSystem, result.FromSystem);
+            AssertIdentityNamePairWithDeactivatedStatus(entity.ToItSystem, result.ToSystem);
+
+            var contractDto = Assert.Single(result.AffectedContracts);
+            AssertIdentityNamePair(affectedContract, contractDto);
+
+            var relationDto = Assert.Single(result.AffectedRelations);
+            AssertIdentityNamePairWithDeactivatedStatus(affectedRelation.ToSystemUsage, relationDto.ToSystemUsage);
+            AssertIdentityNamePairWithDeactivatedStatus(affectedRelation.FromSystemUsage, relationDto.FromSystemUsage);
+            Assert.Equal(affectedRelation.Description, relationDto.Description);
+            AssertIdentityNamePair(affectedRelation.RelationInterface, relationDto.Interface);
+            AssertIdentityNamePair(affectedRelation.UsageFrequency, relationDto.FrequencyType);
+            AssertIdentityNamePair(affectedRelation.AssociatedContract, relationDto.Contract);
+
+            var dprDto = Assert.Single(result.AffectedDataProcessingRegistrations);
+            AssertIdentityNamePair(affectedDpr, dprDto);
+        }
+
+        [Fact]
+        public void Can_MapUnusedSystems()
+        {
+            //Arrange
+            var system1 = CreateSystem();
+            var system2 = CreateSystem();
+
+            //Act
+            var result = _sut.MapUnusedSystems(new List<ItSystem> {system1, system2});
+
+            //Assert
+            var unusedSystems = result.ToList();
+            Assert.Equal(2, unusedSystems.Count);
+            var systemDto1 = Assert.Single(unusedSystems, x => x.Uuid == system1.Uuid);
+            AssertIdentityNamePairWithDeactivatedStatus(system1, systemDto1);
+            var systemDto2 = Assert.Single(unusedSystems, x => x.Uuid == system2.Uuid);
+            AssertIdentityNamePairWithDeactivatedStatus(system2, systemDto2);
+        }
+
+        [Fact]
+        public void Can_MapCommandPermissions()
+        {
+            //Arrange
+            var commandPermission = new CommandPermissionResult(A<string>(), A<bool>());
+
+            var expectedDto = new CommandPermissionResponseDTO
+            {
+                Id = commandPermission.Id,
```

Comment:
you can just use the `A<CommandPermissionResponseDTO>()` because what you are interrested in is that it's response from the mapper that is embedded in the total response. It will save you some lines here.